src/apps/bristleback_apps/bm_rbr/CMakeLists.txt

```diff
@@ -107,6 +108,7 @@ set(LIB_FILES
     ${SRC_DIR}/lib/drivers/w25.cpp
     ${SRC_DIR}/lib/bm_serial/bm_serial.c
     ${SRC_DIR}/lib/memfault/memfault_platform_core_u5.c
+    # ${SRC_DIR}/lib/memfault/memfault_lptim_software_watchdog.c
```

Comment:
I used a software watchdog for debugging and we could enable this, however, if we want it, it will need to be added to all of the builds since it would share the same task as the hardware watchdog. That should also be its own PR since it involves turning on the LPTIM2 for every bsp.

I left it commented out for now as an example, but if it makes it cleaner to remove I can do that.